packages/rest/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:18.04 as base
```

Comment:
I think we can remove the docker files (also the .dockerignore) for now. Maybe later we can publish a rest package for AFJ.